File: C#/Vetores.cs
using System;

namespace atribuiçãoValores
{
    class Atribuição
    {
        static void Main(string[] args)
        {

            //DECLARAÇÃO VARIÁVEIS
            int qtyNumbers;

            //LEITURA INPUTS
            Console.Write("Quantos números você irá digitar? ");
            qtyNumbers = int.Parse(Console.ReadLine());

            //DECLARAÇÃO VETOR
            double[] vet = new double[qtyNumbers];

            //LEITURA VETOR
            for (int i = 0; i < qtyNumbers; i++)
            {
                Console.Write("Digite o número para a posição " + (i + 1) + "ª do vetor: ");
                vet[i] = double.Parse(Console.ReadLine());
            }

            //PRINTANDO VETOR
            Console.WriteLine("\n" + "O vetor digitado foi: ");
            for (int i = 0; i < qtyNumbers; i++)
            {
                Console.WriteLine(vet[i].ToString("F1"));
            }
        }
    }

}

File: java/Vetores.java

//import java.util.Locale;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {

		// DECLARAÇÃO VARIÁVEIS
		Scanner sc = new Scanner(System.in);
		int qtyNumbers;

		// LEITURA INPUTS
		System.out.print("Quantos numeros voce vai digitar? ");
		qtyNumbers = sc.nextInt();

		// DECLARAÇÃO VETOR
		double[] vetNumbers = new double[qtyNumbers];

		// LEITURA VETOR
		for (int i = 0; i < qtyNumbers; i++) {
			System.out.print("Digite um numero: ");
			vetNumbers[i] = sc.nextDouble();
		}

		// PRINT OUTPUTS
		System.out.println("\nNUMEROS DIGITADOS:");
		for (int i = 0; i < qtyNumbers; i++) {
			System.out.println(String.format("%.2f", vetNumbers[i]));
		}

		// Desalocar recursos do Scanner
		sc.close();
	}
}
